Consul + Vault - TLS (#195)
hashicorp/katakoda: consul-secure-tls-vault/step2.md

<!-- How would you feel about moving this before the Consul template step? Then in the Consul template step they can see the cert being rotated 

We could move the configuration into a specific step and explain at the bottom that we are going to automate the config part using consul-template

-->

> Now configure Consul using the `server.json`{{open}}

Reviewer: ```suggestion
Now configure Consul using the `server-tls.json`{{open}}
```

Author: Very good catch, thanks.

configuration file provided with the lab.

Example content:

```
{
  "verify_incoming": true,
  "verify_outgoing": true,
  "verify_server_hostname": true,
  "ca_file": "/opt/consul/agent-certs/ca.crt",
  "cert_file": "/opt/consul/agent-certs/agent.crt",
  "key_file": "/opt/consul/agent-certs/agent.key",
  "auto_encrypt": {
    "allow_tls": true
  }
}
```

To configure TLS encryption for Consul, three files are required:

* `ca_file`   - CA (or intermediate) certificate to verify the identity of the other nodes.
* `cert_file` - Consul agent public certificate
* `key_file`  - Consul agent private key

> For the first Consul startup you will use the certificate generated earlier.

Reviewer 2: ```suggestion
For the first Consul startup, you will use the certificate generated earlier.
```

> You can manually create the files using the output of the command before now stored in `certs.txt` or you can use the following commands to extract the content from the file and place it into the right file and location.

Reviewer 2: ```suggestion
Use the following commands to extract the two certificates and private key from the `certs.txt` and place them into the right file and location.
```
